### PR TITLE
Adding CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by Supun94**
PR for issue #3

### CODE REVIEW

**Comment by dafky2000** [comment]
at first glance there are a couple things i would like to change. will review and discuss changes later on today! thank you!!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+## How to contribute to simplectemplate
+
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
+Please note we have a Pull Request Process, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+* Creating a new branch off of the develop branch
+* All changes to the code should have appropriate unit tests
```
Comment (dafky2000) [inline CONTRIBUTING.md]: Lets delete this one above for now until #2 is done.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+* All code should match PSR-2 style
```
Comment (dafky2000) [inline CONTRIBUTING.md]: Please remove above for now, I will update with my fussy coding style after #6

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+* Submit a pull request again develop as the base branch
+* All travis checks will need to pass
```
Comment (dafky2000) [inline CONTRIBUTING.md]: Again, above is not applicable right now. I have created #12 and will add after that is complete.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+#### **Did you find a bug?**
+
+
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/dafky2000/simplectemplate/issues).
+
+
+* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/dafky2000/simplectemplate/issues/new). Be sure to include a **title and clear description**, as much relevant information as possible, and a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+
+
+#### **Did you write a patch that fixes a bug?**
```
Comment (dafky2000) [inline CONTRIBUTING.md:20]: Let's remove this section, maybe change the title of "Pull Request Process" to something else - "Fixing issues & Pull request process"?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+
+* Open a new GitHub pull request with the patch.
+
+
+* Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.
+
+
+#### **Do you intend to add a new feature or change an existing one?**
```
Comment (dafky2000) [inline CONTRIBUTING.md]: Let's get rid of this section, I don't mind getting feature requests (as long as they aren't duplicated) as a method of getting some initial feedback.

**Review by dafky2000 — CHANGES_REQUESTED** [review]
@Supun94, I have made some notes about my review. Overall it is great! Just a few small changes if you don't mind :)

**Comment by Supun94** [comment]
@dafky2000 I have done the necessary changes. Please review :)

**Review by dafky2000 — APPROVED** [review]
Thank you @Supun94!

**Pull request merged**